src/modDesc.xml

```diff
@@ -120,4 +131,16 @@ Changelog ${version}:
 			<ru>Очистить контракты</ru>
 		</text>
 	</l10n>
+    <actions>
+        <action name="MENU_EXTRA_3" category="SYSTEM" axisType="HALF"  ignoreComboMask="true"/>
+        <action name="SC_Con"       category="PLAYER" axisType="HALF"  ignoreComboMask="false"/>
+    </actions>  
+    <inputBinding>
+        <actionBinding action="MENU_EXTRA_3">
+            <binding device="KB_MOUSE_DEFAULT" input="KEY_s" index="1"/>
+        </actionBinding>
+        <actionBinding action="SC_Con" >
+            <binding device="KB_MOUSE_DEFAULT" input="KEY_lshift KEY_c" />
+        </actionBinding>
```

Comment:
only used when debug == true. Will print info on all avaiablel missions to the log

Reply:
Can you please remove this?
I know it's only used if `debug == true` but it will always register the actions and bindings on every user `inputBinding.xml` so that they cannot use the same keyboard keys for anything else.
Try using a console commands instead.
`addConsoleCommand`

Reply:
good hint - addConsole Command was much easier than going via an action key!